udplog/_kafka.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Ralph Meijer.
+# See LICENSE for details.
+
+"""
+Kafka support.
+
+This provides a Twisted based publisher to a topic in a Kafka cluster.
+"""
+from __future__ import division, absolute_import
+
+import socket
+
+from kafka import KafkaClient, SimpleProducer
```

Comment:
It is async. See a producer is created with [async=True](https://github.com/mochi/udplog/pull/10/files#diff-6595c40ee48d307c5c7f824de7f3e6d9R47). So every time when an message is submitted with [send_messages](https://github.com/mochi/udplog/pull/10/files#diff-6595c40ee48d307c5c7f824de7f3e6d9R39) it is actually queued to an producer internal [Queue](https://docs.python.org/2/library/queue.html). If the queue is full then `send_messages` fails with exception which is catched and logged in [dispatcher](https://github.com/mochi/udplog/blob/master/udplog/twisted.py#L224-L227).